Microsoft.Toolkit.Uwp.SampleApp/SamplePages/HamburgerMenu/HamburgerMenuCode.bind

```diff
@@ -54,6 +54,7 @@
                                 DisplayMode="@[DisplayMode:Enum:SplitViewDisplayMode.CompactInline]"
                                 CompactPaneLength="@[CompactPaneLength:Slider:48:10-80]"
                                 HamburgerHeight="@[HamburgerHeight:Slider:48:10-80]"
+                                HamburgerVisibility="@[HamburgerVisibility:Enum:Visibility.Visible]"
```

Comment:
@deltakosh  That was my first idea as well. But then I saw that all other properties of the hamburger button only have the "Hamburger" prefix (HamburgerHeight, HamburgerWidth, HamburgerMargin, they all affect the button and not the icon itself). So for having naming consistency, I chose to use "HamburgerVisibility". But I agree that HamburgerButtonVisibility is the better name. Shall I change it?